### PR TITLE
Fix help for multi word commands

### PR DESCRIPTION
**Pull request by vlasenko**
This PR fixes the behavior of `help` cmd for the multi-word commands, so that `yarn help workspaces list` worked as expected.

### CODE REVIEW

File: packages/plugin-essentials/sources/commands/help.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
 // eslint-disable-next-line arca/no-default-export
 export default (clipanion: any) => clipanion
 
-  .command(`help [command]`)
+  .command(`help [...command]`)
   .describe(`print the program usage`)
 
   .action(async ({command, ...rest}: {command: string | null, rest: {[key: string]: any}}) => {
```
Comment (arcanis) [inline packages/plugin-essentials/sources/commands/help.ts]:
```suggestion
   .action(async ({command, ...rest}: {command: Array<string>, rest: {[key: string]: any}}) => {
```

File: packages/plugin-essentials/sources/commands/help.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
     if (command) {
```
Comment (arcanis) [inline packages/plugin-essentials/sources/commands/help.ts]:
```suggestion
     if (command.length > 0) {
```

**Comment by vlasenko** [comment]
@arcanis Tests fail, because `build:cli` fails:
```
Error: Module not found: Error: Can't resolve '@berry/pnp/lib/hook' in 'd:\a\1\s\packages\berry-pnp\sources'
    at factory.create (d:\a\1\s\.yarn\cache\webpack-npm-4.28.4-4d1ea327cf8dc1716553cd79f3b4769e71ae06a15f58c4878401ee28d4b610f6.zip\node_modules\webpack\lib\Compilation.js:821:10)
    at factory (d:\a\1\s\.yarn\cache\webpack-npm-4.28.4-4d1ea327cf8dc1716553cd79f3b4769e71ae06a15f58c4878401ee28d4b610f6.zip\node_modules\webpack\lib\NormalModuleFactory.js:397:22)
    at resolver (d:\a\1\s\.yarn\cache\webpack-npm-4.28.4-4d1ea327cf8dc1716553cd79f3b4769e71ae06a15f58c4878401ee28d4b610f6.zip\node_modules\webpack\lib\NormalModuleFactory.js:130:21)
...
```
but it fails not because of my changes, I think

**Comment by arcanis** [comment]
👌

**Pull request merged**